docs/configuration.rst

```diff
@@ -63,4 +75,10 @@ from the setting.
 If you are concerned having `md5` and `sha1` enabled, you can always adjust the setting after
 the migration is done and remove those two checksum types from the allowed list. After modifying
 the setting, you likely will need to run ``pulpcore-manager handle-artifact-checksums`` to remove
-unsupported checksums from the database, or Pulp will refuse to start.
\ No newline at end of file
+unsupported checksums from the database, or Pulp will refuse to start.
+
+.. note::
+
+    If you experience Pulp 3 workers timing out during the migration, consider making them more
+    tolerant to delayed PostgreSQL writes by increasing the Pulp 3 ``WORKER_TTL`` setting. Typically
+    a value of 300 seconds has worked well even during large migration workloads.
```

Comment:
Depending on whether we backport this patch, a note about which Pulpcore version is required to do this might make sense.

Reply:
+1 to mention 3.11. 
I believe we do not plan to backport it. Katello will likely patch the hardcoded constant we used to have in any older releases they need.

Reply:
I'll add the 3.11 reference. We can't backport in pulpcore because it's a feature.